### DISCUSSION TITLE
Why the final score could be positive?

### DISCUSSION THREAD

**Discussion by Legend94rz** [PaddlePaddle/PaddleSpatial#120] [BaiduKDDCup2022] [answered]
If I understand correctly, the score of the perfect submit is zero, right?

**Comment by zhoujingbo** [comment]
Yes, the perfect score should be zero, which means the model precisely predicts the result. 

Thanks a lot.

**Comment by ramber1836** [comment, marked as answer]
This bug has been fixed.
